pages/fundamentals/acknowledgements.html

```diff
@@ -204,39 +199,37 @@ <h1>Acknowledgments</h1>
 darkCavalier11,
 jordancurve,
 Aleksandr Shalimov,
-etc.
+기타 다른 모든 분들.
 </p>
 
 <p>
-I'm sorry if I forgot mentioning somebody in above lists.
-There are so many kind and creative gophers in the Go community
-that I must have missed out on someone.
+만약 제가 위에서 언급하는 것을 잊은 분이 있다면 죄송합니다.
+Go 커뮤니티에는 친절하고 독창성있는 고퍼들(Gophers)이 너무 많아서 몇몇 분들을 놓쳤을 수도 있습니다.
 </p>
 
 <p>
-Thanks to the authors of the
+go101.org 웹사이트를 지원해주신
 <a href="https://getbootstrap.com/docs/3.3/">Bootstrap framework</a>,
 <a href="https://jquery.com">jQuery library</a>,
-<a href="https://github.com/google/code-prettify">code prettify</a> and
-<a href="https://prismjs.com/">prism syntax highlighting</a> JavaSript packages,
-for supporting the go101.org website.
-Thanks to the authors of the
-<a href="https://github.com/bmaupin/go-epub">go-epub</a> and
-<a href="https://calibre-ebook.com/">calibre</a> projects
-for building Go 101 ebooks.
+<a href="https://github.com/google/code-prettify">code prettify</a> 그리고
+<a href="https://prismjs.com/">prism syntax highlighting</a> JavaScript 패키지
+작성자에게 감사드립니다.
+
+Go 101 전자책 제작을 위해주신
```

Comment:
```suggestion
Go 101 전자책 제작을 맡아주신
```

오타인가요? 그냥 제가 모르는 표현일수도 있지만...

Reply:
요기도 수정하였슴다